src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -814,50 +867,55 @@ class device {
         }
         ostr << "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
         ostr << "Temperature(C)\n";
-        // use get_pretty for Temperature and Electrical since the driver may rail unsupported values high
         ostr << std::setw(16) << "PCB TOP FRONT" << std::setw(16) << "PCB TOP REAR" << std::setw(16) << "PCB BTM FRONT" << std::endl;
-        ostr << std::setw(16) << sensor_tree::get_pretty<unsigned short>( "board.physical.thermal.pcb.top_front" )
-             << std::setw(16) << sensor_tree::get_pretty<unsigned short>( "board.physical.thermal.pcb.top_rear"  )
-             << std::setw(16) << sensor_tree::get_pretty<unsigned short>( "board.physical.thermal.pcb.btm_front" ) << std::endl;
+        ostr << std::setw(16) << sensor_tree::get( "board.physical.thermal.pcb.top_front", -1 )
```

Comment:
why get_pretty() -> get()?

Reply:
get_pretty was converting ints to hex, but now we can directly get hex so no need to convert!

Reply:
seems like we need get_pretty for when the value returned by sysfs_get is invalid. So if a node is invalid, it returns int_max instead of 0 or NULL. I can either change these back to get_pretty, or change get func to do what get pretty is doing